src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -119,9 +119,10 @@ private function parseShortOption($token)
     private function parseShortOptionSet($name)
     {
         $len = \strlen($name);
+        $encoding = \mb_detect_encoding($name);
```

Comment:
`mb_detect_encoding` might return `false`, see e.g. https://github.com/symfony/symfony/blob/e81285249b780a11ed209a79fa77c1f6ea6da67b/src/Symfony/Component/Console/Helper/Helper.php#L68.  The code should keep relying on `$name[$i]` in this case